kura/examples/org.eclipse.kura.example.container.signature.validation/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,7 @@ Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
 Service-Component: OSGI-INF/*.xml,
  OSGI-INF/dummy_signature_validation.xml
 Bundle-ActivationPolicy: lazy
-Import-Package: org.eclipse.kura.container.orchestration;version="1.3.0",
+Import-Package: org.eclipse.kura.container.orchestration;version="[1.3,2.0)",
  org.eclipse.kura.container.signature;version="[1.0,2.0)",
```

Comment:
Since you are implementing the `org.eclipse.kura.container.signature` providerType interface, you have to set the range `[1.0,1.1)`.

Reply:
Addressed in 0058fc5b75c8a12bc6bbd3763b168fbb078b98fe